roles/fio-distributed/templates/client.yaml

```diff
@@ -38,6 +38,9 @@ spec:
         command: ["/bin/sh", "-c"]
         args:
           - "cat /tmp/host/hosts;
+{% if fiod.prefill is sameas true %}
+             cat /tmp/fio/fiojob-prefill; mkdir -p /tmp/fiod-{{ uuid }}/fiojob-prefill; python /opt/snafu/run_snafu.py -t fio -H /tmp/host/hosts -j /tmp/fio/fiojob-prefill -s 1 -d /tmp/fiod-{{ uuid }}/fiojob-prefill;
```

Comment:
if we're not interested in pre-fill results maybe we dont need to invoke snafu ? otherwise it'll skew results as these documents would still have same identifiers

Reply:
@aakarshg why would you want to invoke fio a different way? Seems awfully counterproductive.

Reply:
concern is around how do we throw away these prefill documents ? since we're using the container, we'll need to figure out a way to not index these documents, there's 2 ways - either we disable indexing these documents, by unsetting es vars and then resetting them or we can just invoke fio directly. IMHO the right way to do this would be to add another flag that identifies documents generated by prefill job, so that when we do aggregation or comparison, we can filter by that identifier and moving forward if we're even interested in those documents we can still do that.

Reply:
It was my understanding based on the job below, there would be no results created?

Reply:
the job below that was newly added is exactly the same as the normal one, so i'm not sure what you mean by there'll be no results created

Reply:
ack - i thought based on our discussion with @acalhounRH the fill job would be ignored. I wonder if we should not pass the `directory` ?